openlibrary/plugins/upstream/borrow.py

```diff
@@ -12,7 +12,7 @@
 
 from infogami import config
 from infogami.utils import delegate
-from infogami.utils.view import public
+from infogami.utils.view import public, render_template
 from infogami.infobase.utils import parse_datetime
 
 from utils import render_template
```

Comment:
Should this `render_template` import still be included? Seems like a name conflict...

Reply:
Did the second commit fix that or do you still see an issue.

Reply:
Yup, looks better now. I'll review the rest of the branch.